app/Http/Controllers/DashboardController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Modules\HR\Entities\Applicant;
+use Carbon\Carbon;
+
+class DashboardController extends Controller
+{
+    public function index()
+    {
+        $todayCount = Applicant::whereDate('created_at', '=', now())
+    ->count();
+        
+        $record = Applicant::select(\DB::raw("COUNT(*) as count"), \DB::raw("MONTHNAME(created_at) as month"), \DB::raw("DATE(created_at) as date"))
+    ->where('created_at', '>', Carbon::today()->subDay(7))
+    ->groupBY('date')
+    ->orderBy('date', 'ASC')
+    ->get();
```

Comment:
Correct the formatting
```suggestion
        $todayCount = Applicant::whereDate('created_at', '=', now())
            ->count();
        
        $record = Applicant::select(\DB::raw("COUNT(*) as count"), \DB::raw("MONTHNAME(created_at) as month"), \DB::raw("DATE(created_at) as date"))
            ->where('created_at', '>', Carbon::today()->subDay(7))
            ->groupBY('date')
            ->orderBy('date', 'ASC')
            ->get();
```